src/Microsoft.ML.StandardTrainers/FactorizationMachine/FactorizationMachineCatalog.cs

```diff
@@ -14,19 +14,19 @@ namespace Microsoft.ML
     public static class FactorizationMachineExtensions
     {
         /// <summary>
-        /// Predict a target using a field-aware factorization machine algorithm.
+        /// Create an <see cref="FieldAwareFactorizationMachineTrainer"/>, which predicts a target using a field-aware factorization machine trained over boolean label data.
```

Comment:
>an [](start = 19, length = 2)

Check with @natk if we need "an" for my PRs there was no "an" or "a"

Reply:
Let me just follow you.

---
In reply to: [277106453](https://github.com/dotnet/machinelearning/pull/3374#discussion_r277106453) [](ancestors = 277106453)